### DISCUSSION TITLE
Conditional env file.

### DISCUSSION THREAD

**Discussion by PrawiraGenestonlia** [prisma/prisma#5371] [Q&A]
if I want to define conditional dbpath given different NODE_ENV, how do i do it?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
